### PR TITLE
build(package): Replace SASS with SASSC in the patternfly-sass gem

### PR DESCRIPTION
**Pull request by skateman**
SASS became EOL in favor of SASSC http://sass.logdown.com/posts/7081811

cc @simaishi @himdel @epwinchell @jeff-phillips-18

### CODE REVIEW

**Review by jeff-phillips-18 — APPROVED** [review]

**Comment by ezr-ondrej** [comment]
Sorry for duplicating, I haven't noticed your PR, I opened #1199. I believe you're missing the precision fixes here. Those probably work because of `sass-rails` aliasing, but we shouldn't relly on that as we don't require `sass-rails`.
I'll go ahead and close my PR? Or can I somehow help to move things forward?

File: patternfly-sass.gemspec [hunk 1]
```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/Patternfly/patternfly'
   s.license  = 'Apache-2.0'
 
-  s.add_runtime_dependency 'sass', '~> 3.4.15'
+  s.add_runtime_dependency 'sassc', '~> 3.6.1'
```
Comment (ezr-ondrej) [inline patternfly-sass.gemspec:14]: sassc is `2.2.1` ATM newest version, so this is IMHO wrong.
```suggestion
  s.add_runtime_dependency 'sassc', '~> 2.0'
```

**Comment by ezr-ondrej** [comment]
@jeff-phillips-18 could we progress with this? :) anything I can do? my PR #1199 is green and I believe ready.

**Comment by redallen** [comment]
Closed in #1199

**Pull request closed without merging**